### PR TITLE
fix: Change ffmpeg to post process at priority 20

### PR DESCRIPTION
**Pull request by rrauenza**
- Closes https://github.com/cypress-io/cypress/issues/21585

### User facing changelog
 
Run the post processing ffmpeg at lower priority, similar to when capturing

### Additional details

When running concurrent cicd, the ffmpeg consumes a LOT of CPU.  My experience showed 600%.

### How has the user experience changed?
No

### PR Tasks

- [n/a ] Have tests been added/updated?
- [x] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [n/a ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ n/a] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Comment by rrauenza** [comment]
PR: https://github.com/cypress-io/cypress/issues/21585

**Comment by BlueWinds** [comment]
Yeah, that's why we push it off until after the tests complete - and then when there's nothing else competing for CPU, allow it to process as quickly as it can. "One thread per CPU, process as quickly as possible" is the intended behavior here.

Generally you don't want to be running multiple instances of Cypress on the same machine at the same time. The preferred pattern is to use docker containers for parallelization - and then use Docker's facilities to manage resources.

**Comment by BlueWinds** [comment]
On further consideration, I think I'm fine with the change - if ffmpeg is the only thing running, then it won't matter if priority is set lower, it won't slow down video processing.

I updated the PR with latest develop in order to kick off the automated tests - once they're green I'll verify that they haven't been slowed down and we can merge in.

File: packages/server/lib/video_capture.ts [hunk 1]
```diff
@@ -283,7 +283,9 @@ export async function process (name, cname, videoCompression, ffmpegchaptersConf
     debug('processing video from %s to %s video compression %o',
       name, cname, videoCompression)
 
-    const command = ffmpeg()
+    const command = ffmpeg(
+      priority: 20,
```
Comment (BlueWinds) [inline packages/server/lib/video_capture.ts:287]:
```suggestion
      priority: 20
```

File: packages/server/lib/video_capture.ts [hunk 2]
```diff
@@ -283,7 +283,9 @@ export async function process (name, cname, videoCompression, ffmpegchaptersConf
     debug('processing video from %s to %s video compression %o',
       name, cname, videoCompression)
 
-    const command = ffmpeg()
+    const command = ffmpeg(
+      priority: 20
+    )
```
Comment (BlueWinds) [inline packages/server/lib/video_capture.ts:288]:
```suggestion
    const command = ffmpeg({
      priority: 20
    })
```

**Comment by BlueWinds** [comment]
@rrauenza - Looks mostly good, just need to clean up the dangling comma so it'll pass linting.

**Comment by rrauenza** [comment]
@BlueWinds Actually... that lint error was really good because I didn't put the priority: 20 into a {}.  I don't understand the lint error... there *was* a trailing comma, but the code was wrong.   Fix pushed just now.

**Review by BlueWinds — APPROVED** [review]

**Comment by BlueWinds** [comment]
@rrauenza - Looks good to me. We're holding off merging anything else in before the 10.0.0 release tomorrow, but will try and get this in for 10.0.1.

**Pull request merged**